dashboard/config/routes.rb

```diff
@@ -187,6 +181,8 @@
     end
   end
 
+  get "/gallery", to: redirect("/projects")
```

Comment:
nit: I think this should redirect to /projects/public, since /gallery was the old public gallery (right?) and /projects/public is the new public gallery